src/test/java/com/commercetools/sync/producttypes/utils/producttypeactionutils/BuildAttributeDefinitionUpdateActionsTest.java

```diff
@@ -231,7 +232,11 @@ public void buildAttributesUpdateActions_WithOneExtraAttribute_ShouldBuildAddAtt
         );
 
         assertThat(updateActions).containsExactly(
-            AddAttributeDefinition.of(ATTRIBUTE_DEFINITION_D)
+            AddAttributeDefinition.of(AttributeDefinitionDraftBuilder
```

Comment:
Is there a difference or you just checking the factory method is working?

Reply:
because of 2 things:

1. The previously used `AddAttributeDefinition#of(AttributeDefinition)` is now deprecated.
2. The created `ATTRIBUTE_DEFINITION_D` gets by default set values for `attributeConstraint`, `required` and `searchable` fields. Which is not the case for when creating an `attributeDefinitionDraft`. And the action itself is built from a draft now after a change as opposed to from a AttributeDefinition like it was before.